src/clj/game/cards/programs.clj

```diff
@@ -877,4 +877,48 @@
                    :counter-cost [:power 3]
                    :once :per-turn
                    :msg "gain [Click][Click]"
-                   :effect (effect (gain :click 2))}]}})
+                   :effect (effect (gain :click 2))}]}
+
+   "Wari"
+   {:events {:successful-run
+             {:interactive (req )
+              :delayed-completion true
+              :req (req (and (= target :hq)
+                             (first-successful-run-on-server? state :hq)
+                             (some #(and (ice? %)
+                                         (not (rezzed? %)))
+                                   (all-installed state :corp))))
+              :effect (effect
+                       (continue-ability
+                        {:prompt "Use Wari?"
+                         :choices ["Yes" "No"]
+                         :delayed-completion true
+                         :effect (effect 
+                                  (continue-ability
+                                   (if (= target "Yes")
```

Comment:
This is a little messy. I think you meant for the `if` to be before the `continue-ability`, as in, if the Runner clicks Yes, then continue this ability, otherwise trigger `effect-completed`. This approach *might* have the same overall effect, but for clarity we should switch it around. You'll have to use `(req` instead of `(effect`.